### PR TITLE
forum: fixing the cite button, no need to refresh anymore

### LINKED ISSUE #192: Forum: cite functionality is broken
The page has to be refreshed for the "citation" to appear.
This is probably caused by the emoji picker

### PR DESCRIPTION
**Pull request by Elfikurr**
Fixes #192 

Hello,

This PR will fix the cite button on the forum.
It refreshes the emoji-picker innerHTML attribute in order to match what is on the $post_textarea side.
The users no longer need to refresh the page.

I have tested on my end, and it appears to be working correctly.

Thank you.

### CODE REVIEW

**Review by seirl — APPROVED** [review]

**Comment by seirl** [comment]
You need to rebase that on master apparently.

File: prologin/forum/templates/forum/thread_posts.html [hunk 1]
```diff
@@ -203,9 +203,12 @@ <h3 id="preview">{% trans "Preview" context "Preview, the noun (title)" %}</h3>
       $('.btn-cite').click(function(e) {
         e.preventDefault();
         var cite_url = $(this).attr('href');
+        console.log(cite_url);
```
Comment (zopieux) [inline prologin/forum/templates/forum/thread_posts.html]: Remove your debug lines :-)

**Review by zopieux — APPROVED** [review]
Thanks! One nit.

File: prologin/forum/templates/forum/thread_posts.html (hunk 1 above)
Comment (Elfikurr) [reply prologin/forum/templates/forum/thread_posts.html]: Oops :D

**Pull request merged**